extension/scripts/lib/tests/index.js

```diff
@@ -2,7 +2,8 @@
 /* globals debug, chrome */
 
 const tests = {
-	thirdParty: require('./blocking-3rd-party')
+	thirdParty: require('./blocking-3rd-party'),
+	scrolling : require('./scroll')
```

Comment:
Theses tests are for the DevTools script rather than the client I should have added a comment to make that clear.